### PR TITLE
[stable10] Remove config value 'appstoreurl' - the appstore is long g…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…one - long live the marketplace

backport of #30400 
Fixes https://github.com/owncloud/core/issues/30397

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -723,26 +723,6 @@
...
-/**
- * Apps
- *
- * Options for the Apps folder, Apps store, and App code checker.
- */
-
-/**
- * The URL of the appstore to use.
- */
-'appstoreurl' => 'https://api.owncloud.com/v1',
```
Comment (PVince81) [inline config/config.sample.php]: Market app still uses this:
```
apps3/side/market/lib/MarketService.php:70:             $storeUrl = $config->getSystemValue('appstoreurl', 'https://marketplace.owncloud.com');
```

but I'm fine removing this anyway

Comment (DeepDiver1975) [reply config/config.sample.php]: yes - but we better document this in the market app - or just drop it there. It was never usefull

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by phil-davis** [comment]
drone restarted...

**Comment by DeepDiver1975** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**